cli/run.sh

```diff
@@ -1 +1,4 @@
-python -m cli.text2embeddings --s3 $EMBEDDINGS_INPUT_PREFIX $INDEXER_INPUT_PREFIX --device=cpu
\ No newline at end of file
+#!/bin/bash
+set -e
```

Comment:
Out of interest is this best practice or is this necessary as we were successfully passing in the prefixes prior to this 'set -e' change?

Reply:
those 2 lines are currently "best practice & not really necessary", but as soon as another line turns up in this script we'll be happy they're there.